mlflow/genai/judges/utils.py

```diff
@@ -32,7 +32,9 @@ def _sanitize_justification(justification: str) -> str:
     return justification.replace("Let's think step by step. ", "")
 
 
-def invoke_judge_model(model_uri: str, prompt: str, assessment_name: str) -> Feedback:
+def invoke_judge_model(
+    model_uri: str, prompt: str, assessment_name: str, num_retries: int = 10
```

Comment:
@harupy I increased retry count to 10 by default, since 7 was ~ 30 seconds. 10 is > 1 minute